Things.md

```diff
@@ -0,0 +1,126 @@
+  
+### Table of Contents  
+1. [GroupHoldr](#groupholdr)
+    1. [Getting Groups](#getting-groups)
+2. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+    2. [Opacity](#opacity)
+
+# GroupHoldr  
+
+GroupHoldr is a way to store similarly typed information, such as Things, based on a shared property. Each Thing has a `groupType` string property that determines what group it's considered to be in. Some examples from `FSM` (FullScreenmario) are:
+
+* Text
+* Character
+* Solid
+* Scenery 
+
+GroupHoldr contains an Array for each of the groups; each Array contains all the Things of that type currently in the game. Things are added to their respective group when added to the game, and removed when they die. For example, in `FSM`, solid objects like floors and blocks would be stored in the **Solid** group. 
+
+### Getting Groups  
+The groups are accessible both by static name and via passing in a String:
+
+```javascript
+// Returns the Solid group
+this.getGroup("Solid");
+
+// Returns the first Solid
+this.getGroup("Solid")[0];
+this.getSolid(0);
+```
+
+More can be read on GroupHoldr on its [Readme](https://github.com/FullScreenShenanigans/GroupHoldr/blob/master/README.md).
+
+# Manipulating Things  
+
+## Shifting Things  
+A Thing's placement on the screen is based on its `top`, `right`, `bottom`, and `left` properties. These define the bounding box for a Thing. A Thing is basically just a rectangle, with its height, width, and location defined by these properties.  
+The coordinate system in which these Things are placed has an origin at the top left corner (0,0). Things are also scaled from these raw measurements to actual in-game measurements using a predefined `unitsize` (i.e. `unitsize` of 1 means no scaling).    
+There are a number of ways to move Things, all of which are done by manipulating these 4 properties.  
+
+#### Shift  
+The first is to use the shift functions `shiftVert` and `shiftHoriz`. These move the Thing up and down or left and right.
+```javascript
+// Shifts a Thing vertically by 1.
+shiftVert(thing, 1);
+
+// Shifts a Thing horizontally by 1. 
+shiftHoriz(thing, 1);
+```
+#### Set  
+The next is to use a `set` function (`setTop`, `setRight`, `setBottom`, or `setLeft`). These move the Thing to line up with the bounding property.
+```javascript
+// Sets the top bound of a Thing to 10. 
+setTop(thing, 10);
+
+// Sets the left bound of a Thing to 10. 
+setLeft(thing, 10);
+
+// Sets the bottom bound of a Thing to 10. 
+setBottom(thing, 10);
+
+// Sets the right bound of a Thing to 10. 
+setRight(thing, 10);
+```  
+#### Midpoints  
+The next is to set the Thing's midpoint using `setMidX`, `setMidY`, and `setMid` (the latter of which sets both the `x` and `y` midpoints). These functions make it so the Thing is centered on the given `x` and `y`.  
```

Comment:
You say "The next is to" in a bunch of these. Try using starters like "You can" instead.